tools/moth.py

```diff
@@ -328,15 +334,35 @@ def __init__(self, path, seed):
         self.path = path
         self.seed = seed
         self.pointvals = []
+        self.catmod = None
+
+        try:
+            catmod = SourceFileLoader(
+                'catmod',
+                os.path.join(path, 'category.py')).load_module()
+            assert all([
+                hasattr(catmod, 'make'),
+                hasattr(catmod, 'points'),
+                type(catmod.points) is list,
+            ])
+            self.catmod = catmod
+            self.pointvals.extend(catmod.points)
```

Comment:
Nope. The presence of `category.py` means you're generating the category. The category author doesn't get to then drop in other things piecemeal and cause us to have to sort out how to deal with there being two 20-point puzzles.

Note that this doesn't prevent the author from calling `puzzle.read_directory()` in their make function.